packages/terra-signature/src/Signature.jsx

```diff
@@ -3,18 +3,23 @@ import PropTypes from 'prop-types';
 import 'terra-base/lib/baseStyles';
 import styles from './Signature.scss';
 
-const LineWidths = {
+const LINEWIDTHS = {
   EXTRA_FINE: 1,
   FINE: 2,
   MEDIUM: 4,
   HEAVY: 6,
 };
+
+const MOUSECODES = {
```

Comment:
Should we underscore Line widths and Mouse codes? 

```
LINE_WIDTHS
MOUSE_CODES
```

Reply:
I was going with consistency here with constants in other packages like hookshot, which don't use underscores.

Reply:
I'm cool with no underscore all caps.